src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -1,19 +1,23 @@
-const convertInclude = ({ href, url, ...rest }) => {
+const convertInclude = ({ href, url, type, ...rest }) => {
   const supportedTypes = {
+    indepthtoolkit: 'idt1',
     idt2: 'idt2',
-    include: 'include',
+    include: 'vj',
   };
-  const type = href.split('/')[1];
+  const typeExtraction = href.split('/')[1];
```

Comment:
Should we have some defensive code here to catch an edge case where the href value could be:
- `'sometext'` - this has no `/` in it
- `null` or `undefined`

Just return null if `href.split('/').length < 2`? Would need an extra test too